src/main/java/google/calendar/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore Gradle project-specific cache directory
```

Comment:
Why do we need a gitignore file in your package?

Reply:
removed.